content/sdk/quickstarts/cpp/_index.md

```diff
@@ -24,4 +24,6 @@ Find the finalized code for these quickstarts on {{< externalLink link="https://
 - A Qrypt Account. {{< externalLink link="https://portal.qrypt.com/register" text="Create an account for free" >}}.
 
 ## Quickstarts
-1. [Distributed key generation](keygendistributed/)
+1. {{< externalLink link="https://github.com/QryptInc/qrypt-security-quickstarts-cpp" text="Distributed key generation" >}}.
```

Comment:
Saw external links syntax like {{< externalLink link="https://github.com/QryptInc/qrypt-security-quickstarts-cpp" text="GitHub" >}}. on the md file https://github.com/QryptInc/QryptInc.github.io/blob/move-quickstart-instructions-to-github/content/sdk/quickstarts/cpp/_index.md but I assume they would display normally on html files and the website.